packages/template-retail-react-app/app/hooks/use-add-to-cart-modal.js

```diff
@@ -65,16 +66,15 @@ export const AddToCartModal = () => {
     const totalQuantity = itemsAdded.reduce((acc, {quantity}) => acc + quantity, 0)
 
     return (
-        <Modal size={size} isOpen={isOpen} onClose={onClose}>
+        <Modal size={size} isOpen={isOpen} onClose={onClose} scrollBehavior="inside" isCentered>
             <ModalOverlay />
             <ModalContent
                 margin="0"
-                marginTop={{md: '8', lg: '200px'}}
-                overflow="hidden"
                 borderRadius={{base: 'none', md: 'base'}}
                 bgColor="gray.50"
+                dialogProps={{maxHeight: '90vh'}}
```

Comment:
There is `containerProps`, but I don't see `dialogProps`.. that's why the max-height is not being applied.

So I'm ok with removing it. Seems like passing `isCentered` prop to Modal already has the intended effect.
```suggestion
```